tests/e2e/pageobjects/ide/Terminal.ts

```diff
@@ -124,7 +124,7 @@ export class Terminal {
     }
 
     private getTerminalTabCssLocator(tabTitle: string): string {
-        return `li[title='${tabTitle}']`;
+        return `li[title='${tabTitle}'] div.theia-tab-icon-label`;
```

Comment:
I guess it can break some cases, [here](https://github.com/eclipse/che/blob/4b09aeb9ebdc8a565f630f3e5f3bca413d1f7142/tests/e2e/pageobjects/ide/Terminal.ts#L65), for example

Reply:
Let's wait on test results https://codeready-workspaces-jenkins.rhev-ci-vms.eng.rdu2.redhat.com/job/Multiuser-Che-PR-check-E2E-Happy-path-tests-against-k8s/3617/